extensions/binding/org.eclipse.smarthome.binding.tradfri/src/main/java/org/eclipse/smarthome/binding/tradfri/GatewayConfig.java

```diff
@@ -17,8 +17,14 @@
     public static final String HOST = "host";
     public static final String PORT = "port";
     public static final String CODE = "code";
+    
+    public static final String USERNAME = "username";
```

Comment:
Could we go for the same wording as IKEA? This would be IDENTITY and PRE_SHARED_KEY. This would make it much simpler for others to understand the code.